lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -134,16 +143,22 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
         ),
       ),
       'twitter' => array(
```

Comment:
Hi @angaither @ngjo @mikefantini ! Exploring this a little more, adding this to the "Stuff You Need" section is a little more difficult than I thought. Since there isn't a set format (bullet points vs. regular text) and the length will be different for each of the campaigns, it's hard to append to the end of this string: 
![screen shot 2016-06-13 at 12 00 16 pm](https://cloud.githubusercontent.com/assets/9019452/16014280/99e28fe0-315e-11e6-9ba4-3a5a23a9654b.png)

I would recommend that we add another share about in this "STEP 2: PLAN IT" section or customizing the share bar at top and guiding the users there to share their particular links. Thoughts?